codegen/manifest.json

```diff
@@ -126,6 +126,20 @@
         {
             "type": "prefix",
             "patterns": ["CONTACT"],
```

Comment:
@RaphaelHeiniger this is my best understanding from the manual which contact keywords do not have the ID, MPP, and A->G options.